tests/integration_tests/tests/agentless_tests/shared_resource/test_relationship_running_workflow.py

```diff
@@ -11,24 +11,25 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+import pytest
 
 from integration_tests import AgentlessTestCase
-from integration_tests.tests.utils import get_resource as resource
 
 
+@pytest.mark.usefixtures('mock_workflows_plugin')
 class TestRelationshipRunningRemoteWorkflow(AgentlessTestCase):
     shared_resource_blueprint = """
 tosca_definitions_version: cloudify_dsl_1_3
 
 imports:
     - cloudify/types/types.yaml
-    - wf--blueprint:mock_workflows
+    - plugin:mock_workflows
```

Comment:
hmm no, this makes it a different test, the point was using something from a different blueprint

Reply:
Done and updated